src/popup/pages/Send/index.js

```diff
@@ -151,8 +162,24 @@ class SendPage extends React.Component {
         toAddressTitle=getLanguage('toAddress')
         confirmTitle = getLanguage('sendDetail')
         confirmToAddressTitle = getLanguage('toAddress')
-
         isWithdraw = true
+
+        warnBeforeSending = async () => {
+          const ownAddresses = this.state.allAccounts.commonList
```

Comment:
this races the WALLET_GET_ALL_ACCOUNT, but that should be fast compared to the human interaction leading to this, right?

Reply:
Yeah :shrug:

Reply:
Fixed in https://github.com/oasisprotocol/oasis-wallet-ext/pull/236/commits/d5af6aa7cc9539a6484392947ceb360b15bfcdce